entrypoint.sh

```diff
@@ -12,10 +12,27 @@ fi
 
 # create the static files to be used by the Django admin portal
 python manage.py collectstatic --noinput
+# find and delete any remaining migration files before creating new ones
+find . -path "*/migrations/*.py" -not -name "__init__.py" -delete
+find . -path "*/migrations/*.pyc"  -delete
+# migrate our current THR schema to the MySQL database
+python manage.py makemigrations
+python manage.py migrate
+
+# build ES index
+python manage.py search_index --rebuild -f
 
 # create a superuser defined by the environment variables
-echo "from django.contrib.auth.models import User;
+# Fix: 'Manager isn't available' error
+# https://stackoverflow.com/a/67530965/4488332
+echo "from django.contrib.auth import get_user_model;
+User = get_user_model();
 User.objects.filter(email='$DJANGO_ADMIN_EMAIL').delete();
 User.objects.create_superuser('$DJANGO_ADMIN_USER', '$DJANGO_ADMIN_EMAIL', '$DJANGO_ADMIN_PASSWORD')" | python manage.py shell
```

Comment:
I tried to do it before using `python manage.py createsuperuser --noinput || true` but it's only working with Django 3+ ([here is an old commit where I tried to use it](python manage.py createsuperuser --noinput || true;))
[This answer confirms it](https://stackoverflow.com/a/59467533/4488332)

Reply:
I meant why not use a custom Django Command, like in the [next answer](https://stackoverflow.com/a/42491469) to that StackOverflow question?  it's better to avoid executing Python by evaluating a string because it makes the Python accessible to your IDE, linter etc

Reply:
That's a good idea, I'll do that

Reply:
I added `create_superuser` command
```
$ python manage.py create_superuser --username "$DJANGO_ADMIN_USER" --password "$DJANGO_ADMIN_PASSWORD" --noinput --email "$DJANGO_ADMIN_EMAIL"
Superuser created successfully.
```